cranelift/assembler-x64/meta/src/dsl.rs

```diff
@@ -4,10 +4,13 @@
 //! - compact--i.e., define an x64 instruction on a single line, and
 //! - a close-to-direct mapping of what we read in the x64 reference manual.
 
+pub(crate) mod custom;
```

Comment:
```suggestion
mod custom;
```

No need to publicize this; we re-publish the types we want to be public in the `pub use custom::...` below.